src/pages/tezos/customizebot.js

```diff
@@ -487,6 +485,10 @@ const Customizer = () => {
   };
 
   const isBrowser = typeof window !== 'undefined';
+  const canvas = useRef(null);
```

Comment:
@manangouhari  Please remove irrelevant commits.

Reply:
Oops, I didn't realise this was pushed too. Fixed it.